ChangeLog.md

```diff
@@ -1,5 +1,9 @@
 # ChangeLog for file-embed
 
+## 0.0.15.1
```

Comment:
A minor version bump would bring this to 0.0.16, necessary by the PVP since this adds a new identifier.

Reply:
Fixed